logic/scripts/update_token_insight.py

```diff
@@ -321,9 +333,46 @@ def fetch_ogn_transactions():
             db.session.add(tx)
             db.session.commit()
 
+# Fetches wallet balance from API and stores that to DB
+def fetch_wallet_balance(wallet):
+    print "Checking the balance of wallet %s" % (
+        wallet,
+    )
+
+    url = "http://api.ethplorer.io/getAddressInfo/%s" % (wallet)
```

Comment:
How often will we call this API? Just double checking you made sure it won't go over quota.
https://github.com/EverexIO/Ethplorer/wiki/Ethplorer-API#freekey-limits

Reply:
Yes, The script is run every 10 minutes. So, we are on the safe side, at least for now.